batch.go

```diff
@@ -0,0 +1,134 @@
+package pgxmock
+
+import (
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
+)
+
+// BatchElement helps to create batches for testing
+type BatchElement struct {
+	sql          string
+	rewrittenSQL string
+	args         []interface{}
+}
+
+// NewBatchElement creates new query that will be queued in batch.
+// Function accepts sql string and optionally arguments
+func NewBatchElement(sql string, args ...interface{}) *BatchElement {
```

Comment:
```suggestion
func NewBatchQuery(sql string, args ...interface{}) *BatchElement {
```
According to previous suggestion